styles/components/_form.scss

```diff
@@ -26,7 +26,7 @@ textarea,
   color: inherit;
   border-radius: 3px;
   font: {
-    family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-family: "Noto Sans", sans-serif;
```

Comment:
There's a typo here, it should be "family" not "font-family" (SASS turns this into "font-font-family" if you look in your web dev panel), but font family is set by the base file, so it can just be removed here.